docs/octopus-cloud/static-ip.md

```diff
@@ -1,11 +1,13 @@
 ---
-title: Static IP address
+title: Static IP Address
 position: 20
 description: How to find the list of static IP addresses for your Octopus Cloud instance
 ---
 
 The Octopus Cloud is a multi-tenant service with several static IP addresses shared among customers in the same Azure region.
 
-Each Azure region uses a pool of 10 static IP addresses, and the static IP for an Octopus Cloud instance will use one from the pool and may change to another static IP from the pool under certain situations.
+Each Azure region uses a pool of 10 static IP addresses, and the static IP for an Octopus Cloud instance will use one from the pool, and may change to another static IP from the pool under certain situations.
```

Comment:
Slight tweak:
```suggestion
Each Azure region uses a pool of 10 static IP addresses. The static IP for an Octopus Cloud instance will use one from the pool and may change to another static IP from the pool under certain situations.
```

Reply:
It might be useful to add the scenarios (even a general hint) that change the IP address.